install/roles/qpc_server/tasks/main.yml

```diff
@@ -10,6 +10,13 @@
     server_port: "{{ ansible_env['SERVER_PORT'] | default('9443') }}"
   when: server_port is not defined
 
+- name: open port in firewall
+  iptables:
+    chain: INPUT
+    destination_port: "{{server_port}}"
+    protocol: tcp
+    jump: ACCEPT
+  ignore_errors: yes
```

Comment:
I think we should add `become: true` here.

Reply:
Also, I feel like we should have an `open_port` flag or something so that they can turn on/off this functionality